docs/customer_management/cp_configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Configure Customer Portal to fit the needs of your business.
+edition: experience
+---
+
+# Customer Portal configuration
+
+You can overwrite the default configuration of the Customer Portal
+to fit its capabilities to the unique needs of your business.
+
+## `corporate` SiteAccess
+
+The predefined `corporate` SiteAccess in `corporate_group`
+(configured in `config/packages/ibexa.yaml`) serves the Customer Portal.
+If you need a multisite setup with multiple Customer Portals,
+add any additional SiteAccesses to `corporate_group`.
+
+## Customer identifier
+
+`ibexa_default_settings.yaml` contains setting that indicate which Content Types are treated like customers:
+
+```yaml
+ibexa:
+    system:
+        default:
+            user_content_type_identifier: ['user', 'customer']
+
+```
+
+You can override these settings if you have other Content Types
+that should be treated as customers in the Back Office and Customer Portal.
+When viewing such Content in the Back Office and Customer Portal, you will be able to see e.g. the assigned Policies.
+
+## Roles and Policies
+
+You can easily add custom roles to your installation
+by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+
+```yaml
+parameters:
+    ibexa.site_access.config.default.corporate_accounts.roles:
+        admin: Company Admin
+        buyer: Company Buyer
+        custom_role: Company Assistant
+```
+
+## Content Types names
+
+To change names of default Content Types, you need to define them in the configuration
```

Comment:
That doesnt make sense, name changing is done in UI, what you could do is assign what content types should be used to describe `Company` and `Member` in Domain layer.
But those configs are done under `ibexa_corporate_account` key, like shown in `\Ibexa\Bundle\CorporateAccount\DependencyInjection\Configuration`.

So if you already have `company` CT in your system, and you dont want to change that identifier, then you should configure CA in following way.
```
ibexa_corporate_account:
    content_type_mappings:
        company: your_ct_identifier
```

Be aware tho, that you have to adjust migration files by yourself if you are in that position.